datadog/fwprovider/framework_provider.go

```diff
@@ -183,6 +183,23 @@ func (p *FrameworkProvider) Schema(_ context.Context, _ provider.SchemaRequest,
 				Description: "The HTTP request maximum retry number. Defaults to 3.",
 			},
 		},
+		Blocks: map[string]schema.Block{
+			"default_tags": schema.ListNestedBlock{
+				Validators: []validator.List{
+					listvalidator.SizeAtMost(1),
```

Comment:
Why the list can only have size 1 ?

Reply:
because we only want `tags` in there, nothing else - the actual list of tags can have more that one tag.
It's a bit strange (I could have gone for `default_tags{}` instead of `default_tags{tags{}}`) but it's the way it's done for the AWS provider, so it's how the feature was requested and probably how the users expect this to work.